src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+    <div class="stars stars--1">
```

Comment:
```suggestion
    <div class="stars stars--0">
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
    </div>
    
    <div class="stars stars--1">
```
1'st point in checklist